services/frontend/src/scenes/Projects/Projects.scss

```diff
@@ -0,0 +1,115 @@
+@import "styles/_colors.scss";
+@import "styles/_dimens.scss";
+@import "styles/_mixins.scss";
+
+.projects {
+    display: flex;
+    flex-direction: column;
+    height: calc(100% - #{$navbar-height});
+    overflow: auto;
+}
+
+.projects-header {
+    border-bottom: 1px solid $color-text-secondary;
+    display: flex;
+    flex-shrink: 0;
+    justify-content: center;
+    padding: $default-spacing;
+}
+
+.filter-button {
+    background-color: transparent;
+    border: none;
+    border-radius: $default-border-radius;
+    box-shadow: none;
+    color: $color-text-secondary;
+    margin: 0 16px;
+    padding: $default-spacing/2 $default-spacing*1.5;
+    text-transform: none;
+
+    &.filter-button--active {
+        background-color: $color-primary;
+        color: $color-light;
+    }
+}
+
+.projects-list {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    padding: $default-spacing * 2;
+
+    .project-card {
+        margin: $default-spacing 0;
+    }
+}
+
+.project-card {
+    display: flex;
+    flex-shrink: 0;
+    max-width: $default-spacing * 34;
+
+    .project-card-content {
+        display: flex;
+        padding: $default-spacing*1.5 $default-spacing*2;
+
+        .project-card-active-section {
+            align-self: flex-start;
+            display: flex;
+            flex: 1;
+        }
+
+        .project-card-content-section {
+            flex: 3;
+
+            .project-card-name {
```

Comment:
I would change this to `project-card-title` since in reality this is an `<h3>` for the card.

Reply:
But is the fact that it's an `h3` really relevant? e.g. I could change the underlying element to be just a `<p>` tag, but that wouldn't change the fact that the information being encoded by this particular element is the _name_ of the project, not the 'title' of the card. It just so happens that, visually, it _is_ the name of the card.

Reply:
Fair, it's sort of a correlation vs causation situation. It's not a title because it's an h3 but titles tend to take on heading tags. 

That said, even ignoring the tag I would probably still call this a card `title` instead of a `name` since `title` more effectively describes it's role within the card.

Reply:
But that's the point I'm trying to argue; I'm naming these classes in accordance with the data they _represent_, not how they are _visually_ presented.

To more clearly make my point, would you equally make the argument that `project-card-description` should instead be called `project-card-subtitle`, since that is what the element is presented visually as?

Reply:
Alright, I see where you're coming from. With layouts I tend towards naming elements what they represent on the page and use the tags as a general indicator of what those elements represent. I concede this isn't particularly consistent and comes down more to the developer choice.

I find these sorts of names slightly more intuitive to develop views with but I'm alright with basing it off the data itself. This seems like a good thing to add to the Nuclino section on CSS/Sass.